tool-base/src/main/kotlin/io/spine/tools/code/Language.kt

```diff
@@ -114,6 +114,7 @@ public abstract class Language internal constructor(
  *
  * Supports double-slash comments (`// <comment body>`).
```

Comment:
I would like to avoid supporting both methods, as the slash-asterisk covers both possible cases. Since we're generating code here, not parsing it (and thus don't require full language feature support), I'd accept such simplifications.

It's true that double-slash comments are common. However, our little model cannot and should not reflect all possible aspects of the domain.

Let's discuss this further in a call if needed.